src/ui-test/tests/codelenses/jbang.test.ts

```diff
@@ -0,0 +1,149 @@
+import * as path from 'path';
+import { variables } from '../../variables';
+import {
+    after,
+    before,
+    CodeLens,
+    DebugToolbar,
+    EditorTabNotFound,
+    EditorView,
+    error,
+    errors,
+    repeat,
+    Repeat,
+    resources,
+    TextEditor,
+    until,
+    VSBrowser,
+    workspaces
+} from 'vscode-uitests-tooling';
+
+const file = 'JBang.yaml';
```

Comment:
what about to provide more specific file name than `jbang.test.ts` to usecase we are testing?